### DISCUSSION TITLE
AppImage error - missing libatk-1.0.so

### DISCUSSION THREAD

**Discussion by zaunere** [mistweaverco/bananas#22] [Q&A] [answered]
Hello,

When running the AppImage:

 ./bananas_x86_64.AppImage
/tmp/.mount_bananaGgPWhd/bananas: error while loading shared libraries: libatk-1.0.so.0: cannot open shared object file: No such file or directory

I don't have libatk but would think I don't need it.

md5sum bananas_x86_64.AppImage 
74e710ec40213220256ded53d85b4aa0  bananas_x86_64.AppImage

Linux fffr 6.1.106-116.188.amzn2023.x86_64 #1 SMP PREEMPT_DYNAMIC Tue Aug 27 07:00:36 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

Thanks,

Hans

**Comment by gorillamoe** [comment, marked as answer]
That's not bundled into the app right now, but needed for interaction with UI.

Might want to try installing it, or I might find a way to bundle that also.

There's an open issue at the electron repo, where they discuss the missing documentation about that requirement.
